Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -234,15 +200,15 @@ extension PlaceCarouselViewController: InAppNotificationToastDelegate {
 
 extension PlaceCarouselViewController: LocationMonitorDelegate {
     func locationMonitor(_ locationMonitor: LocationMonitor, didUpdateLocation location: CLLocation) {
-        if !isNoLocationAlertPresented {
+        if !Prompts.isNoLocationAlertPresented {
             updateLocation(location: location)
             return
         }
 
         // If we don't dismiss the location alert, the detail view controller (called from
         // updateLocation) does not get presented.
         dismiss(animated: true) {
-            self.isNoLocationAlertPresented = false
+            Prompts.isNoLocationAlertPresented = false
```

Comment:
It manages it if the close button is pressed but not if it's dismissed in code (as it is here). We could probably fix this by subclassing UIAlertViewController & overriding `viewWillDisappear` or whatever.